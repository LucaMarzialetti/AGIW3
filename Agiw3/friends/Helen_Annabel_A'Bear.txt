--url_fb https://www.facebook.com/helen.abear.1
--friends
VM11972:3 Sarah A'Bear
VM11972:3 Steve Lawrence
VM11972:3 David Harry
VM11972:3 Marta Okrojek
VM11972:3 Daniel Jackson
VM11972:3 Annie Thomas
VM11972:3 Adam Philip
VM11972:3 Viola Szekely
VM11972:3 Anna Lundy
VM11972:3 Amy Coombes
VM11972:3 Christopher Brough
VM11972:3 Owen Llewelyn
VM11972:3 Jab Miah
VM11972:3 Raghad Brough
VM11972:3 Joanna Hackett
VM11972:3 Harry H Peez
VM11972:3 Daniella Elson
VM11972:3 Derry Thomas O'Neil
VM11972:3 Alex McCarney
VM11972:3 Emily De La Ronde-Wilton
VM11972:3 Gwenda Wozencraft
VM11972:3 Luke Beasley
VM11972:3 Alex Nicholas
VM11972:3 Jessica Rivers
VM11972:3 Alex Ong
VM11972:3 Lucy Bayliss
VM11972:3 Vicky Johnson
VM11972:3 Imogen Evans
VM11972:3 Elin Davies
VM11972:3 Abbey Jones
VM11972:3 Kat Rosen
VM11972:3 Victoria Louise Gunstone
VM11972:3 Aaron Hulston
VM11972:3 LucyVictoria Gilmore
VM11972:3 Chris Burchill
VM11972:3 Natalie Jayne Edwards
VM11972:3 Sophie Kendrick
VM11972:3 Arnaud Ritter
VM11972:3 Vanya Ivanova
VM11972:3 Rupert Andrew
VM11972:3 Mike Farnham
VM11972:3 Peter Donnison
VM11972:3 Ryan Bayliss
VM11972:3 Tom Surtees
VM11972:3 James Cromwell
VM11972:3 Delyth Wozencraft
VM11972:3 Rebecca Louise
VM11972:3 Nikki Rees
VM11972:3 Danielle Emily Pinocci
VM11972:3 Shelley Evans
VM11972:3 Sara Cunningham
VM11972:3 Lewis Collins
VM11972:3 Martha-Jane Powell
VM11972:3 Ellen Lundy
VM11972:3 Janika Cartwright
VM11972:3 Joe Lingard
VM11972:3 Charlie Elson
VM11972:3 Kelsey Jones
VM11972:3 Ed Mills
VM11972:3 Holly Lloyd
VM11972:3 Matt Ingman
VM11972:3 Ffion Grosvenor
VM11972:3 Oliver Coombes
VM11972:3 Jat Mehmi
VM11972:3 Jude A'Bear
VM11972:3 Chris Nash
VM11972:3 Holly Louise Mcgillivray
VM11972:3 Lloyd Davies
VM11972:3 Mollie Williams
VM11972:3 Sali Roberts
VM11972:3 Sian Taylor
VM11972:3 Adam Alsop
VM11972:3 Adam James
VM11972:3 Daf Roberts
VM11972:3 Thomas Plant
VM11972:3 Siôn Richards
VM11972:3 Zoe Thompson
VM11972:3 Jess Nelms
VM11972:3 Lily Isabelle Tomkins
VM11972:3 Nigel Jamal Brink
VM11972:3 Conor Neal
VM11972:3 Zoe Bladen
VM11972:3 Ellis Hopgood
VM11972:3 Owain Lloyd
VM11972:3 Andrea Chapman
VM11972:3 Joey Anderson
VM11972:3 Carwyn Lloyd Owen
VM11972:3 Adam Hooton
VM11972:3 Melissa Hellio
VM11972:3 James Selvey
VM11972:3 Igor Sasha Bernardi
VM11972:3 Charlotte Rebecca
VM11972:3 James 'Truman' Thurman
VM11972:3 Siân Parry
VM11972:3 Hywel Stoddart
VM11972:3 Nerys Hammond
VM11972:3 Satnam Sangha
VM11972:3 Tom Ray Jackson
VM11972:3 Tom Lord
VM11972:3 Dan Humpage
VM11972:3 Chintu Utnihc
VM11972:3 Andrew Chopper Jones
VM11972:3 Steven Van Der Lammie
VM11972:3 Katie Marie Hughson
VM11972:3 Josh Slade
VM11972:3 Helen Carter
VM11972:3 Holli Heath
VM11972:3 Hannah Lloyd
VM11972:3 Lara Camilla Wilkinson
VM11972:3 Ronen Achrak
VM11972:3 Ashley Sutherland
VM11972:3 Matt Curran Whitburn
VM11972:3 Adam Addison
VM11972:3 Charlie Edwards
VM11972:3 Tom Dickinson
VM11972:3 Chloé Flascher
VM11972:3 Sarah Davies
VM11972:3 David Lewis
VM11972:3 Samsher Singh Bhachu
VM11972:3 Jake Astley
VM11972:3 Elin Wozencraft
VM11972:3 Ben Davies
VM11972:3 Carla Marie Ahmadi
VM11972:3 Rahul Singh
VM11972:3 Veli Noguera Parcet
VM11972:3 Bethan Greaves
VM11972:3 Matthew Sinky Sinclair
VM11972:3 Sally Thomas
VM11972:3 Jaz Yehia
VM11972:3 Harri Beehan
VM11972:3 Emily Ffion Nash
VM11972:3 Sean Michael
VM11972:3 Sarah-Jane Phillips
VM11972:3 Josh Davies
VM11972:3 Chris Archer
VM11972:3 David Rees Evans
VM11972:3 Oliver Weerasuriya
VM11972:3 Conor Rafferty
VM11972:3 Sophie Stephens
VM11972:3 Natasha Bunclark
VM11972:3 Jane Cross
VM11972:3 Rhian Hammond
VM11972:3 Jenny Ireland
VM11972:3 Sarah Newport
VM11972:3 Hannah Templeman
VM11972:3 Chris Earnshaw
VM11972:3 Alex Jones
VM11972:3 Dan Jenkins
VM11972:3 Julia Barry
VM11972:3 Bec Davies
VM11972:3 Stephanie Bacon
VM11972:3 James Cox
VM11972:3 Cherri Douglas
VM11972:3 Jessica Treacher
VM11972:3 Carly Johnson
VM11972:3 Hollie Dinwoodie
VM11972:3 Seamus McKimm
VM11972:3 Gemma Beer
VM11972:3 James Gowen-Smith
VM11972:3 Sammi Ahmed
VM11972:3 Aaron Azu
VM11972:3 Tom Gosling
VM11972:3 Gemma Pugh
VM11972:3 Rhys Bagnall
VM11972:3 Mohammed Qasim
VM11972:3 David Evans
VM11972:3 India Taylor
VM11972:3 Daniel Lee
VM11972:3 Cheryl Samuel
VM11972:3 Don A'Bear
VM11972:3 Jessica Venn
VM11972:3 Nikhil Shah
VM11972:3 Josh Preddy
VM11972:3 Chrissie Monnelly
VM11972:3 Clare Keates
VM11972:3 Louis Gerrard
VM11972:3 Laura Burnell
VM11972:3 George Stephens
VM11972:3 Beca Tuffnell
VM11972:3 Lloyd Haines
VM11972:3 Adam Stephens
VM11972:3 Adam Edwards
VM11972:3 Nick Hammond
VM11972:3 Simon Lawson
VM11972:3 Liam Mackay
VM11972:3 Alex Bollen
VM11972:3 Jason Sugrue
VM11972:3 Pammor Cemplon
VM11972:3 Carwyn Owen Thomas
VM11972:3 Lewis Hurrell
VM11972:3 Joel Moody
VM11972:3 Victoria JN
VM11972:3 Jordan Bullock
VM11972:3 Dejan Stojanovic
VM11972:3 Lloyd Rhys-Jones
VM11972:3 Matthew Musgrave
VM11972:3 Tom Palmer
VM11972:3 Gareth Matthews
VM11972:3 Rhydian Wyn
VM11972:3 Rory Budina
VM11972:3 Chelsea Holder
VM11972:3 Maximillian Hirschmann
VM11972:3 Kirstin Lloyd
VM11972:3 James Clubb
VM11972:3 James Harrison
VM11972:3 Dafydd Llewelyn
VM11972:3 Tom Rowlands
VM11972:3 Nathan Jones
VM11972:3 Will Bingley
VM11972:3 Rhian Mason
VM11972:3 Jamie Rees
VM11972:3 John Pugh
VM11972:3 Lee Phillips
VM11972:3 Will Cleaton
VM11972:3 Joshua James
VM11972:3 Dave Owen
VM11972:3 Rhys Dando
VM11972:3 Ryan Lawrence
VM11972:3 Shane Nolan
VM11972:3 David Alexander Thomas
VM11972:3 Amrit Chahal
VM11972:3 Mobeen Amin
VM11972:3 Patrick Spillane
VM11972:3 Brett Chatwin
VM11972:3 Tom Miller
VM11972:3 Zoe Haden
VM11972:3 Rhys Watkins
VM11972:3 Lee Jarman
VM11972:3 Natascha Gaut
VM11972:3 Tyler Roberts
VM11972:3 Mared Gwenllian Griffiths
VM11972:3 David Gower
VM11972:3 Chris Jones
VM11972:3 Tom Hagyard
VM11972:3 Theo Foster
VM11972:3 John Kennedy Junior
VM11972:3 Matt Bufton
VM11972:3 Emily Kendrick
VM11972:3 Tom Grbac Milburn
VM11972:3 Will Nixon
VM11972:3 Geraint Steyn
VM11972:3 Kate Louise Morgan
VM11972:3 Selina Baden
VM11972:3 Joe Jordan
VM11972:3 Paul Bishop
VM11972:3 Zac Anthony Palmer
VM11972:3 Jodie Ramplee
VM11972:3 Colette Hazen
VM11972:3 Chazz Obiora
VM11972:3 Jonathan Sims
VM11972:3 Liam Haynes
VM11972:3 Tom Stone
VM11972:3 Eryl Wozencraft
VM11972:3 Peter Graystone
VM11972:3 Michael Gaspar
VM11972:3 Sophie Hopkins
VM11972:3 Sean Carmody
VM11972:3 Lauren Thomas
VM11972:3 Joshua Talbot
VM11972:3 Steven Cox
VM11972:3 Tom Joy
VM11972:3 Thomas Toyne
VM11972:3 Stacey Chesterton
VM11972:3 Tom Price
VM11972:3 Raymond Oliver Luke
VM11972:3 Willow Smallbone
VM11972:3 Ashley Pinches
VM11972:3 Abby Melissa Stephens
VM11972:3 Catherine Jones
VM11972:3 Zac Campbell
VM11972:3 Joe Casella
VM11972:3 Ryan Jenkins
VM11972:3 Steve Fuller
VM11972:3 Michael Wynne
VM11972:3 Josh Wood
VM11972:3 Scotty Pritchard
VM11972:3 Natalie Lane
VM11972:3 Richard Hanson
VM11972:3 Shane Jones
VM11972:3 Dan Thomas
VM11972:3 Aled James
VM11972:3 Thomas Cook
VM11972:3 Danny Evs
VM11972:3 Matt Hunt
VM11972:3 Leroy Smith
VM11972:3 Fiona Lai
VM11972:3 Rob Hurlbutt
VM11972:3 Samantha Louise Jebb
VM11972:3 Stephanie Hughson
VM11972:3 James Jammo Morris
VM11972:3 Adam Cichuta
VM11972:3 Sam Hope
VM11972:3 Nicolas Scotto
VM11972:3 Mike Burchill
VM11972:3 James Frost
VM11972:3 Mark Benton
VM11972:3 Rob Tyzzer
VM11972:3 Daniel Weale
VM11972:3 Rhys Bufton
VM11972:3 Pete McHugh
VM11972:3 Henry G Tonks
VM11972:3 Sophie Elizabeth
VM11972:3 Tom Pollitt
VM11972:3 Medi Mortimer
VM11972:3 Natalie Corten
VM11972:3 Sophie Tompkins
VM11972:3 Lucy Todd
VM11972:3 Priyesh D
VM11972:3 Dannielle Hart
VM11972:3 Dave Coughlan
VM11972:3 Dan Small
VM11972:3 Emily Mowgli Griffiths
VM11972:3 Charlotte Harvey
VM11972:3 Laura Felvus
VM11972:3 Sarah Frances Neill
VM11972:3 Dav Laura
VM11972:3 Lara Burkhalter
VM11972:3 Will Jones
VM11972:3 Vicki Gilvear
VM11972:3 Chloe Slade
VM11972:3 Eleanor Wilson
VM11972:3 Guy Eyal
VM11972:3 Bethan Penrhyn-Jones
VM11972:3 Chris Wheatley
VM11972:3 Lucy Reilly
VM11972:3 Vijay Ladhani
VM11972:3 Chiana Cartwright
VM11972:3 Lucy Davies
VM11972:3 Becky Jones
VM11972:3 Daniel Ruell
VM11972:3 Emma Kelly
VM11972:3 Zoë Williams
VM11972:3 Joe Brophy
VM11972:3 Sali Gwen Owens
VM11972:3 James Mead
VM11972:3 Jonathan Airey
VM11972:3 Juliette Sara Jessell
VM11972:3 Margot Helen Black
VM11972:3 Emma Sophia
VM11972:3 Adam Carlo Papandrea
VM11972:3 Bimba James
VM11972:3 Joe Geen
VM11972:3 Jack Mcphee
VM11972:3 Ross Flay
VM11972:3 Alex Jordan
VM11972:3 Clare Hill
VM11972:3 Vicky Smith
VM11972:3 Liz Jewsbury
VM11972:3 Michael Turner
VM11972:3 Becci Chapman
VM11972:3 Marcus Alexander
VM11972:3 Matthew Bowen
VM11972:3 Dough Cardiff
VM11972:3 Aimee Davies
VM11972:3 Ben Cross
VM11972:3 Francine Ann Wall
VM11972:3 Phunam Patel
VM11972:3 Simon Bufton
VM11972:3 Kiah Louise Bassett
VM11972:3 Sara-Mair Smithies
VM11972:3 Liz Cawthorne
VM11972:3 Shelly Louise
VM11972:3 Gareth Jones
VM11972:3 BinHau Lee
VM11972:3 Sarah Emily
VM11972:3 Jenny Brogdon
VM11972:3 Kieran Brett Cruise
VM11972:3 Tim Kennedy
VM11972:3 Dafydd Oldham
VM11972:3 Tim Swinfen
VM11972:3 Rob Whitham
VM11972:3 Daniel Hunter
VM11972:3 Aly Brooms
VM11972:3 Frances A'Bear
VM11972:3 Andrew Claridge
VM11972:3 Sally Bayliss
VM11972:3 Georgia Waters
VM11972:3 Ruth Osborn Ollman
VM11972:3 Ruth Thomas
VM11972:3 Mark Cooper
VM11972:3 Lee Lello
VM11972:3 Charlotte Drew
VM11972:3 Conall Gribben
VM11972:3 Kirsty Doman
VM11972:3 Louise Williams
VM11972:3 Kate Taylor
VM11972:3 Zara Davies
VM11972:3 Jamie Kynaston
VM11972:3 Josh Holdaway
VM11972:3 Si Bound
VM11972:3 Emily Walker
VM11972:3 Daniel Higginson
VM11972:3 Sam Anthony Steele
VM11972:3 Simon O'Dwyer
VM11972:3 Cai Lear
VM11972:3 Jonathan Taylor
VM11972:3 Robert Hughes
VM11972:3 Natasha Claire
VM11972:3 Chloe Lorna Davis
VM11972:3 Steven Thetank Davies
VM11972:3 Andrew Carter
VM11972:3 Gareth Davies
VM11972:3 Paul Powell
VM11972:3 Joe Ayoubkhani
VM11972:3 Dean Morgan
VM11972:3 Keelie Lucie Nye
VM11972:3 Shantanu Navlakha
VM11972:3 Sian Acton
VM11972:3 Michael Watkins
VM11972:3 Jimi Price
VM11972:3 Danny Dig Davies
VM11972:3 Aled Rhidian Jones
VM11972:3 Sam James
VM11972:3 Catherine Emily
VM11972:3 Tyrone Mcfadden
VM11972:3 Holly Jarman
VM11972:3 Nick Matthew
VM11972:3 Jade Hawkins
VM11972:3 Rob Evans
VM11972:3 Dylan Kesh Segelov Comley
VM11972:3 Naomi Gillum
VM11972:3 Becky Hughson
VM11972:3 Bryony Bugg
VM11972:3 Matt Evans
VM11972:3 Brett Randy Groves
VM11972:3 Shaun Hopkins
VM11972:3 Tom Simmons
VM11972:3 Anj MJ
VM11972:3 Tom Davies
VM11972:3 Sophie Deakins
VM11972:3 Emily Louise
VM11972:3 Anwen Mair Evans
VM11972:3 Stephen PT Abberley
VM11972:3 Ryan Pritch
VM11972:3 Gemma Biggs
VM11972:3 Kay Powell
VM11972:3 Hud Son
VM11972:3 Gareth Davies
VM11972:3 James Gannon
VM11972:3 Carys Jones
VM11972:3 Nick Ward
VM11972:3 Amy Powell
VM11972:3 Shalini Jagmohan
VM11972:3 Ben Davies
VM11972:3 Abdullah Kahloon
VM11972:3 Matt Williams
VM11972:3 Jade Wicks
VM11972:3 Mark Thomas Powell
VM11972:3 Ceinwen Mcphee
VM11972:3 Rhys Powell
VM11972:3 Heidi Middleton
VM11972:3 Georgina Brooks
VM11972:3 Laura Zissis
VM11972:3 Carys Vaughan
VM11972:3 Beth Mary
VM11972:3 Brock Sandeman
VM11972:3 Natasha Rowe
VM11972:3 Hannah John
VM11972:3 Scott Tyler
VM11972:3 Adam Willey
VM11972:3 Rhian Jones
VM11972:3 Robert Nicholls
VM11972:3 Rhiannon Evans
VM11972:3 Rhys Morgan
VM11972:3 Joy Bayliss
VM11972:3 Joel Bayliss
VM11972:3 Joe Bayliss
VM11972:3 Dave Owen
VM11972:3 Ryan Morten Roberts
VM11972:3 Emily Morgan
VM11972:3 Kif Eyval
VM11972:3 Jonny Mills
VM11972:3 Kirsten Charlotte
VM11972:3 Jessica Jones
VM11972:3 Sarah Fox
VM11972:3 Georgia Pullen
VM11972:3 Joshua Roderick
VM11972:3 Kirsty Bayliss
VM11972:3 Baz Edwards
VM11972:3 Tom Mills
VM11972:3 Sam Lewis
VM11972:3 Cristina Knill
VM11972:3 Adam Johns
VM11972:3 Jacob Baylis
VM11972:3 Beth Drew
VM11972:3 Rachel Dobb
VM11972:3 Stacey Louise Powell
VM11972:3 James Evans
VM11972:3 Heulwen Thomas
VM11972:3 Jayde Lush
VM11972:3 Siobhan Wood
VM11972:3 Giz Morgan
VM11972:3 Sadie Lewis
VM11972:3 Murray Beehan
VM11972:3 Hannah Mills
VM11972:3 Andrew Morgan
VM11972:3 Sian Acton
VM11972:3 Rob Hughes
VM11972:3 Jonny Price
VM11972:3 Anna Bowen
VM11972:3 Aj Powell
VM11972:3 George Wozencraft
VM11972:3 Phill Morgan
VM11972:3 Simon Boss Johnson
VM11972:3 Jo Parkinson
VM11972:3 Zoë Evans
VM11972:3 Gareth Wood
VM11972:3 Cazy A'Bear
VM11972:3 Louise Jones
VM11972:3 Elle Isle
VM11972:3 Danni Williams
VM11972:3 Christian Jones
VM11972:3 Aneurin Alexanda Frayne
VM11972:3 David Ramplee
VM11972:3 Daniel Stokesie
VM11972:3 Phil Draper
VM11972:3 Nathaniel Payne
VM11972:3 Rose Jackson Taylor
VM11972:3 Fearnley Evison
VM11972:3 Jonny Samuel
VM11972:3 Nathan Davies
VM11972:3 Gwyneth Lewis
VM11972:3 Luke Ruell
VM11972:3 Olivia Hughes
VM11972:3 Katie Louise Rowlands
VM11972:3 Lottie Hopkins
VM11972:3 Polly Kendrick
VM11972:3 Laura Jones
VM11972:3 Alice Hammond
VM11972:3 Tilly Alcayna
VM11972:3 Stephen Alexander Jones
VM11972:3 Wilna Venter
VM11972:3 George Williams
VM11972:3 Katy Matilda Neo
VM11972:3 Tom Hopkins
VM11972:3 James Thomas
VM11972:3 Hannah Bunford-Jones
VM11972:3 Ryan Parker
VM11972:3 Adam Rees
VM11972:3 Emily Price
VM11972:3 Phil Brown
VM11972:3 Jay Evans
VM11972:3 Daniel Vinny Walker
VM11972:3 Caryn Grosvenor
VM11972:3 Steve Jones
VM11972:3 Lynwen Thomas
VM11972:3 Corrina Williams
VM11972:3 Sean Beehan
VM11972:3 John Cozens
VM11972:3 Joe Wooldridge
VM11972:3 Ian James Powell
VM11972:3 Aled Evans
VM11972:3 Huw Thomas
VM11972:3 Billy Field
VM11972:3 Rosie Knight