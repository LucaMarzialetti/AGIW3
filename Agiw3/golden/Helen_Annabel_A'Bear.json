{
    "PATTERN": {
        "tel": [
            "+44 121 232 1389",
            "+44 121 232 1389"
        ],
        "addr": [],
        "email": ["helenannable@eversheds.com"]
    },
    "NER": {
        "LOC": [
            "Estonia",
            "Austria",
            "China",
            "Minworth",
            "United Kingdom",
            "Midlands"
        ],
        "ORG": [
            "Eversheds International Eversheds Global Global English Global Français Global Español Global",
            "Eversheds International",
            "Eversheds International",
            "Eversheds LLP",
            "Mitchells & Butlers Retail Limited",
            "Österreich Belgium China",
            "Midpoint Park",
            "Hozelock Group",
            "Unipart Group of Companies"
        ],
        "PER": [
            "Helen Annable",
            "Partner",
            "Iftkhar Ahmed"
        ]
    },
    "url": "http://www.eversheds.com/global/en/who/people/index.page?person=en/Annable_Helen"
}